docs/api/advanced/index.rst

```diff
@@ -0,0 +1,11 @@
+Advanced Usage
+==============
+
+The following sections describe advanced usage of the library.
+
+
+.. toctree::
+   :maxdepth: 4
+
+   mcstatus.address
+   mcstatus.dns
```

Comment:
Do we really want to make these a part of the public API? These are purely internal tools which don't really have anything to do with obtaining status of minecraft servers. I would rather avoid making these public as it forces us to keep them backwards compatible. I'd put these on the internal page instead.

Reply:
I thought that `mcstatus.dns` is a part of Public API, maybe I just remember #203 wrong. Although, in the #203 description, it's clearly mentioned that this is a part of Public API.

I agree with `Address`.

Reply:
> I thought that mcstatus.dns is a part of Public API, maybe I just remember https://github.com/py-mine/mcstatus/pull/203 wrong. Although, in the https://github.com/py-mine/mcstatus/pull/203 description, it's clearly mentioned that this is a part of Public API.

Fair, I might've originally intended it to be a part of the public API, however I'm not sure if it actually should be. I'd vote for moving it into the internal, and if someone really wants it they can still technically use it as an internal feature, without us guaranteeing backwards compatibility there.

Reply:
Removed `Address` from here in [this force push](https://github.com/py-mine/mcstatus/compare/afb0702d1c9fda079879cb0fb2a0b1ef92fb54f7..f33d6f5e6c80269ef9072380bd2966a4ca44843e).

Reply:
Let's also remove the dns module, as I said, it probably shouldn't be a part of the public API.

Reply:
I would like to hear @kevinkjt2000's mind also.
https://discord.com/channels/936788458939224094/938623566964981802/1061536418175918141